docs/source/specs/typespec/main.tsp

```diff
@@ -163,12 +163,12 @@ namespace Workspaces {
         @doc("The next link in pagination")
         @example("/api/rbac/v2/(resource)/?limit=10&offset=10")
         @format("uri")
-        next: string;
+        next?: string;
```

Comment:
I _think_ what we'd want for these is to make them nullable, since the keys should always exists, where this would imply the key may be optional? Something like this would ensure it's always there, but make the spec update to `nullable: true`:

```suggestion
        next: string | null;
```